zerver/lib/send_email.py

```diff
@@ -31,6 +31,7 @@ def build_email(template_prefix, to_user_id=None, to_email=None, from_name=None,
         'realm_name_in_notifications': False,
         'support_email': FromAddress.SUPPORT,
         'verbose_support_offers': settings.VERBOSE_SUPPORT_OFFERS,
+        'server_uri': settings.SERVER_URI,
```

Comment:
OK, but I don't think we want to be using `server_uri` in emails in general, because most of the time we really want to make sure everything is linking back to the same subdomain.  How about we declare `email_image_base_uri` here, that includes the `static/images/emails/` part?  That seems much more safely declaring what we mean.

BTW, Is this function only used for test code, or is the comment at its top wrong?